src/parser.rs

```diff
@@ -3898,6 +3898,14 @@ pub fn parse_call(tokens: &[Token], index: &mut usize) -> (Call, Option<JaktErro
                             index,
                             ExpressionKind::ExpressionWithoutAssignment,
                         );
+
+                        if let Some(jakt_error) = &err {
```

Comment:
we probably want to fix the warnings for `jakt_error` not being used. Guessing this should be passed along to `error`?

Reply:
Ah yes, sorry. I'm using rust-analyzer which doesn't give me Clippy warnings atm. I think I'll make an if-else out of this and just move err.